src/github.com/matrix-org/dendrite/clientapi/routing/room_tagging.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2017 Vector Creations Ltd
```

Comment:
You are not required to add Copyright for Vector since you are the creator of this file.
You may skip this line or add your name as the copyright.
/ref commits by @anoadragon453 or me.

Reply:
I'm not sure what `\ref` ing commits mean...?

Reply:
I just meant :refer to commits by ....
:)